addons/web/static/src/js/views/abstract_controller.js

```diff
@@ -119,7 +119,9 @@ var AbstractController = mvc.Controller.extend(ActionMixin, WidgetAdapterMixin,
         if (this._searchPanel) {
             this._searchPanel.on_attach_callback();
         }
-        this.renderer.on_attach_callback();
+        if (!(this.renderer instanceof owl.Component)) {
+            this.renderer.on_attach_callback();
+        }
```

Comment:
could you add a test (with a comment saying that we can remove it as soon as controllers are converted into owl)?